tools/clang/unittests/HLSLExec/LongVectorTestData.h

```diff
@@ -246,46 +246,64 @@ INPUT_SET(InputSet::Default3, true, false, false, false, false, true, true,
           true, false, false);
 END_INPUT_SETS()
 
+#define I16(x) static_cast<int16_t>(x)
 BEGIN_INPUT_SETS(int16_t)
 INPUT_SET(InputSet::Default1, -6, 1, 7, 3, 8, 4, -3, 8, 8, -2);
 INPUT_SET(InputSet::Default2, 5, -6, -3, -2, 9, 3, 1, -3, -7, 2);
 INPUT_SET(InputSet::Default3, 5, -6, -3, -2, 9, 3, 1, -3, -7, 2);
 INPUT_SET(InputSet::BitShiftRhs, 1, 6, 3, 0, 9, 3, 12, 13, 14, 15);
+INPUT_SET(InputSet::Bitwise, -32768, -1, 0, 1, 3, 6, 9, I16(0x5555),
+          I16(0xAAAA), 32767);
 END_INPUT_SETS()
+#undef I16
 
+#define I32(x) static_cast<int32_t>(x)
 BEGIN_INPUT_SETS(int32_t)
 INPUT_SET(InputSet::Default1, -6, 1, 7, 3, 8, 4, -3, 8, 8, -2);
 INPUT_SET(InputSet::Default2, 5, -6, -3, -2, 9, 3, 1, -3, -7, 2);
 INPUT_SET(InputSet::Default3, 5, -6, -3, -2, 9, 3, 1, -3, -7, 2);
 INPUT_SET(InputSet::BitShiftRhs, 1, 6, 3, 0, 9, 3, 30, 31, 32);
+INPUT_SET(InputSet::Bitwise, 0, 1, 3, 6, 9, 12, 13, 14, 15,
+          I32(18446744073709551615));
 END_INPUT_SETS()
+#undef I32
 
+#define I64(x) static_cast<int64_t>(x)
 BEGIN_INPUT_SETS(int64_t)
 INPUT_SET(InputSet::Default1, -6, 11, 7, 3, 8, 4, -3, 8, 8, -2);
 INPUT_SET(InputSet::Default2, 5, -1337, -3, -2, 9, 3, 1, -3, 501, 2);
 INPUT_SET(InputSet::Default3, 5, -1337, -3, -2, 9, 3, 1, -3, 501, 2);
 INPUT_SET(InputSet::BitShiftRhs, 1, 6, 3, 0, 9, 3, 62, 63, 64);
+INPUT_SET(InputSet::Bitwise, -9223372036854775807 - 1, -1, 0, 1, 3, 6, 9,
+          I64(0x5555555555555555), I64(0xAAAAAAAAAAAAAAAA),
```

Comment:
```suggestion
          0x5555555555555555ll, 0xAAAAAAAAAAAAAAAAll,
```

Just in case this option hadn't been considered.